main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/gruntwork-io/aws-nuke/internal/awsnuke"
+	"github.com/gruntwork-io/gruntwork-cli/entrypoint"
+	"github.com/gruntwork-io/gruntwork-cli/logging"
+	"github.com/gruntwork-io/gruntwork-cli/shell"
+	"github.com/urfave/cli"
+)
+
+var VERSION string
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "aws-nuke"
+	app.Author = "Gruntwork <www.gruntwork.io>"
+	app.Version = VERSION
+
+	app.Flags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "non-interactive",
+			Usage: "Assume yes for all interactive prompts",
+		},
+	}
+
+	log := logging.GetLogger("aws-nuke")
+	app.Action = func(c *cli.Context) error {
```

Comment:
Define this in a separate, standalone function. It's more readable and may be reusable in other contexts (e.g., at test time).